met-web/src/components/userManagement/listing/UserManagementListing.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useContext } from 'react';
+import TextField from '@mui/material/TextField';
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import SearchIcon from '@mui/icons-material/Search';
+import { User } from 'models/user';
+import { HeadCell, PaginationOptions } from 'components/common/Table/types';
+import { MetPageGridContainer, PrimaryButton } from 'components/common';
+import { Link } from 'react-router-dom';
+import { Link as MuiLink } from '@mui/material';
```

Comment:
met-web/src/components/userManagement/listing/UserManagementListing.tsx

Do we need to put UserManagementListing in the file name since the folder structure implies it? it feels duplicate

Reply:
We will not import this component as met-web/src/components/userManagement/listing/UserManagementListing.tsx. we import it as met-web/src/components/userManagement/listing and it will be resolved to the index.tsx. So that duplication does not happen when we import in the routes page. However inside index.tsx the component will be imported as import UserManagementListing from './UserManagementListing';
so no duplication

Reply:
Inside that folder UserManagementListing is the clearest way to describe that component and no duplication will happen when we import